src/Tribe/Ajax/Dropdown.php

```diff
@@ -88,6 +88,10 @@ public function search_terms( $search, $page, $args, $source ) {
 			}
 		}
 
+		foreach ( $results as $result ) {
+			$result->text = htmlspecialchars_decode( $result->text );
```

Comment:
Let's make sure we escape scary stuff:

```suggestion
			$result->text = wp_kses_post( htmlspecialchars_decode( $result->text ) );
```